app/views/posts/index.html.erb

```diff
@@ -15,7 +15,7 @@
         <span><%= post.likes_counter%></span>
       </div>
       <ul>
-        <% post.comments.each do |comment| %>
+        <% post.recent_comments.each do |comment| %>
             <strong>User:</strong> <%= comment.user.name %><br>
           <li><%= comment.text %></li>
         <% end %>
```

Comment:
- [ ] [OPTIONAL]
5 most recent comments are not displaying as intended. The total number of comments is rather displaying instead of the 5 most recent comments. Let's endeavor to improve upon this aspect.

![image](https://github.com/Samuelwanza/sema/assets/12745474/cd1bab3d-3ded-422b-919b-90fc0c76a9ca)

![image](https://github.com/Samuelwanza/sema/assets/12745474/0aff48e2-0df4-44b9-b43e-0096798b19ff)